core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -1249,6 +1250,10 @@ private void startIntentBasedOnAction(Intent intent) {
     }
   }
 
+  private void removeIntentAction(Intent intent) {
+    intent.setAction(null);
+  }
```

Comment:
This isn't particularly useful, you can just do `setAction` where this is being called.

Reply:
@abdulwd I think in the future we need this method for reuse. What do you think?

Reply:
@gouri-panda Please remove it, I don't think have any instance in our repository where we employ a function for just setting some var null.

Reply:
@gouri-panda Please remove it, I don't think have any instance in our repository where we employ a function for just setting some var null, it looks overkill to me

Reply:
IMO it is not creating a reusable piece of code nor helping in readability so shouldn't be extracted to a separate function.